src/main/java/fi/helsinki/moodi/service/synchronize/enrich/SisuCourseEnricher.java

```diff
@@ -47,18 +48,14 @@ protected SisuCourseEnricher(StudyRegistryService studyRegistryService) {
     }
 
     public void prefetchCourses(List<String> curIds) {
-        List<String> uniqueSisuIds = new LinkedHashSet<>(curIds).stream()
-            .filter(id -> !StudyRegistryService.isOodiId(id)).collect(Collectors.toList());
+        List<String> uniqueSisuIds = new ArrayList<>(new LinkedHashSet<>(curIds));
         prefetchedCursById = studyRegistryService.getSisuCourseUnitRealisations(uniqueSisuIds).stream()
             .collect(Collectors.toMap(c -> c.realisationId, c -> c));
     }
 
     @Override
     protected SynchronizationItem doEnrich(SynchronizationItem item) {
```

Comment:
Tiketillä oli maininta enrich-prosessien yksinkertaistamisesta uuteena tikettinä, miltä näyttää se, ovatko muutokset tuoneet yksinkertaistamismahdollisuutta?

Reply:
Kirjasin erillisen tiketin siitä: https://jira.it.helsinki.fi/browse/MOODI-149